zulip_botserver/zulip_botserver/input_parameters.py

```diff
@@ -40,4 +39,9 @@ def parse_args() -> argparse.Namespace:
         type=int,
         help='Port on which you want to run the Botserver. (default: %(default)d)'
     )
+    parser.add_argument(
+        '--use-env-vars', '-ev',
```

Comment:
`'-ev'` the short flag with two letters seems unusual to me, could it be just `-e` or maybe `-n`? if those are already taken maybe we can just remove the short flag, the verbose one is not painfully long.

Reply:
Okay sure ```-e``` is fine, will make it ```-e```